test/backup.ts

```diff
@@ -223,6 +225,68 @@ describe('Bigtable/Backup', () => {
     });
   });
 
+  describe('copy', () => {
+    beforeEach(() => {
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
```

Comment:
A few questions
1) What's the reasoning behind the eslint-disable-next-line plus the ts-ignore? In my mind, they kind of cancel each other out, but I'm sure that there's a deeper reason I just don't know here 🙂 
2) Is there a way we can not ignore the linter here?
3) if we're ignoring the linter, can we get away with only the eslint-disable-next-line and whatever the error is it's throwing? Or some kind of ts-expect-error? (I can't remember our exact settings for both eslint and the ts checker)

Reply:
These comments may have been leftover from a copy/paste. They are not needed anymore.